themes/digital.gov/layouts/partials/core/card-legislation.html

```diff
@@ -0,0 +1,34 @@
+{{ with .card_data }}
+  <div class="dg-card-legislation">
+    <div class="usa-card__container usa-card-group usa-card--flag">
+      {{ if .title }}
+        <div class="usa-card__header">
+          <h2 class="dg-featured-resource__text-kicker">Related Policy</h2>
+          <h2 class="usa-card__heading">
+            {{ .title }}
+          </h2>
+        </div>
+      {{ end }}
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img
+            src="{{ $.image }}/legislative-stamp-card-logo.png"
+            alt="{{ .title }}"
```

Comment:
This image is `240x221`, but its loading the full-sized `2121px` image.

Is there a smaller version we can provide?

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/b7ab7e0a-e331-4fb7-ad11-affcb2fbce24)

Reply:
Great catch, I've resized this image to 600x400 and added to AWS and updated the code to use the smaller image.

<img width="1668" alt="Screen Shot 2023-10-30 at 3 01 48 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/052412f1-a3b5-4777-86d9-091288ed988e">